backend/app/migrator/migrator.go

```diff
@@ -24,6 +24,12 @@ type Exporter interface {
 	Export(w io.Writer, siteID string) (int, error)
 }
 
+// Mapper defines interface to convert data in import procedure
+type Mapper interface {
+	URL(url string) string
+	LoadRules(reader io.Reader) error
```

Comment:
Presence of this method implicitly makes Mapper statefull and not thread safe. Practically it is not a big problem as our import/export controllers handling concurrency by itself with its own locking, however I can imagine some use cases in the future this could become a problem. The minimal thing we should do is at least mark (comment) all "unsafe" mapper function as "not thread safe", but eliminating it will be even beter. One way I can think about is to modify LoadRules to return the new Mapper struct